### PR TITLE
LocalFileOutputPlugin uses path_prefix, file_ext and sequence_format config parameters

### PR DESCRIPTION
**Pull request by frsyuki**
* Before: directory + "/" + file_name + ".%03d.%02d" + file_ext
* After: path_prefix + sequence_format + file_ext

### CODE REVIEW

**Comment by frsyuki** [comment]
@nahi Could you review this pull-request?

File: embulk-standards/src/main/java/org/embulk/standards/LocalFileOutputPlugin.java [hunk 1]
```diff
@@ -70,11 +71,9 @@ public TransactionalFileOutput open(TaskSource taskSource, final int processorIn
     {
         PluginTask task = taskSource.loadTask(PluginTask.class);
 
-        // TODO format path using timestamp
-        final String fileName = task.getFileNameFormat();
-
-        final String pathPrefix = task.getDirectory() + File.separator + fileName;
```
Comment (nahi) [inline embulk-standards/src/main/java/org/embulk/standards/LocalFileOutputPlugin.java]: :+1: We're now free from File.separator! (source of environment dependency)

**Comment by nahi** [comment]
:+1:

**Comment by frsyuki** [comment]
@nahi  Thank you! Merged.

**Pull request merged**